README.mdown

```diff
@@ -75,7 +75,7 @@ You should not modify or remove existing configurations in this file.
 #### Using another service
 
 If you use another service than Devise or if you implemented your own authentication service, you can call the following method in a controller to shutdown Intercom on logout (self being the controller instance).
-Be aware that if you call this method before a 'redirect_to' (quite common on logout) it will have no impact as it is impossible to update cookies when you use a redirection.
+**Be aware that if you call this method before a 'redirect_to' (quite common on logout) it will have no impact ** as it is impossible to update cookies when you use a redirection.
```

Comment:
This isn't actually bolding the text, it's making it italic with asterisks around it. You need to get rid if the trialing space before the **